### DISCUSSION TITLE
URL forwarding with /subname

### DISCUSSION THREAD

**Discussion by mpi42** [NginxProxyManager/nginx-proxy-manager#2684] [Q&A]
I am trying to set up a redirect with the NPM.
My goal is to point from a domain with /subname to servers running in the background. Unfortunately subdomains are not possible or the whole thing should run dynamically at some point.

Example: 
Client1 -> name.domain.com/test1 --> server 1 port 80
Client2 -> name.domain.com/test2 --> server 2 port 80
![hsc_x1_show drawio](https://user-images.githubusercontent.com/50237136/225310205-15d94725-77fd-4593-ad42-787fcc60a5ed.png)

How can I implement this in the NPM have now already tried some things in the interface but do not make it. 
Can someone help me here please?

![a1](https://user-images.githubusercontent.com/50237136/225310226-e6aab3da-907f-4485-b6b7-067e3255cae5.PNG)
![a2](https://user-images.githubusercontent.com/50237136/225310249-d02f4b91-f53d-4188-a6eb-0ef527836682.PNG)

Thanks a lot

**Comment by instantdreams** [comment]
It should be like this:
![image](https://user-images.githubusercontent.com/15875093/225760336-9211d745-cd0e-40af-8c37-6ababc0d1ad9.png)

The location should be a forward slash: `/`
The forward location should have the path and include a trailing forward slash: `10.0.0.10/test/`

**Reply by mpi42** [reply]
what if the forwarded location did not have a folder and is only a 10.0.0.10/? 
thank you

**Reply by instantdreams** [reply]
What you're trying to do might need a forward proxy.

**Reply by mpi42** [reply]
> What you're trying to do might need a forward proxy.

how to archive that else? thanks

**Comment by Anupamp-009** [comment]
I want to forward my domain to a specific login form. Please help.

xyz.com to /support/user.php

If I am doing custom location, its not loading user.php
